f8a_worker/process.py

```diff
@@ -356,8 +356,14 @@ def fetch_npm_artifact(name, version, target_dir):
         #      └── package.tgz
         # 3 directories, 6 files
         name_ver = name
+        invalidChars = set(string.punctuation.replace(".", "").replace("-", ""))
```

Comment:
I am not sure if this is going to work. Take a look here: https://docs.npmjs.com/misc/semver#usage

I can see following there:
`5.0.0 - 7.2.3`

None of the characters there is invalid for us, but yet it's a version range and not a specific version.